contrib/testsuites/run-kvm-unit-test.sh

```diff
@@ -34,12 +34,12 @@ while [ true ]; do
             WILDCARD="$1"
             ;;
         "-h"|"--help")
-            echo "Usage: $0 [-h] [--endian ENDIAN] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]"
+            echo "Usage: $0 [-h] [--configure-args=[[--endian ENDIAN] [kvm-unit-test configure arguments]...]] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]"
```

Comment:
Well this is not entirely true, the `--configure-args` accepts only one argument, not multiple ones.

Anyway I have to apologize for making it actually harder than it was. This is a shell script and not python's argparse, so the `=` parsing is not necessary and we can simply take the first argument. So the usage should be:

```
... [--configure-args EXTRA_CONFIGURE_ARGS] ...
```

and the parser is the same as for other args (simple `shift` and get value).

Once again, I'm sorry for confusing you with the `=`, it belonged to argparse which errors out on values starting with `-`.